### ISSUE TITLE
Shader warning in 'SR_Triangles_Object_Space'

### ISSUE THREAD

**Issue by CDiaz-MS** [microsoft/MixedRealityToolkit-Unity#6609]
## Describe the bug
New shader warning after recent modification from #6578.
```
Shader warning in 'SR_Triangles_Object_Space': loop only executes for 1 iteration(s), forcing loop to unroll at line 524 (on d3d11)

Compiling Geometry program
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_FULL_HDR

```

## To reproduce
This warning is printed on import of the new packages

**Comment by Troy-Ferrell** [comment]
@CDiaz-MS  and @keveleigh , this is fixed with merged PR #6578 correct?

**Comment by keveleigh** [comment]
@Troy-Ferrell this one's not, no. Just #6577
